hw3/02-charts.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>02 - Charts</title>
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+    />
+    <link rel="stylesheet" href="/hw3/02-charts.css" />
+    <script src="https://cdn.jsdelivr.net/npm/chart.js@2.8.0"></script>
+    <script defer src="02-charts.js"></script>
+  </head>
+  <body>
+    <main>
+      <div class="'container'">
```

Comment:
Why is the classname surrounded by both single and double quotes?

```suggestion
      <div class="container">
```